### DISCUSSION TITLE
I add media keys but I can't get them to function.

### DISCUSSION THREAD

**Discussion by zepled112** [KinesisCorporation/Adv360-Pro-ZMK#514] [Q&A] [answered]
I will accept if I'm a goofball and it's just a skill issue, but I'll throw kvol, kmute, etc (obviously spelled and done properly) and no matter which version of the media keys I try I can't get them to work. Does anyone have a method that actually works? (doesn't work on linux or windows)

**Comment by ReFil** [comment, marked as answer]
Typically you want the c_vol keys

**Reply by zepled112** [reply]
I was right about only one thing, it was in fact a skill issue. Thanks for taking the time to help me out!!!
